runtime/vm/jvminit.c

```diff
@@ -4369,7 +4369,7 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 #if JAVA_SPEC_VERSION >= 24
 	{
 		IDATA enableYieldPinning = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXYIELDPINNEDVIRTUALTHREADS, NULL);
-		IDATA disableYieldPinning = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXNOYIELDPINNEDVIRTUALTHREADS, NULL);
+		IDATA disableYieldPinning = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXYIELDPINNEDVIRTUALTHREADS, NULL);
```

Comment:
This looks incorrect. Shouldn't it check for `VMOPT_XXNOYIELDPINNEDVIRTUALTHREADS`  ?

Reply:
missed this when c/p